src/report_table.js

```diff
@@ -639,12 +638,17 @@ looker.plugins.visualizations.add({
     // console.log(config)
     var dataTable = new VisPluginTableModel(data, queryResponse, config);
     this.trigger('registerOptions', dataTable.getConfigOptions());
-    buildReportTable(config, dataTable, updateColumnOrder, element);
-
-    // DEBUG OUTPUT AND DONE
-    // console.log('dataTable', dataTable)
-    // console.log('container', document.getElementById('visContainer').parentNode)
-
-    done();
+    buildReportTable(config, dataTable, updateColumnOrder, element)
+      .then(() => {
+        // DEBUG OUTPUT AND DONE
+        // console.log('dataTable', dataTable)
+        // console.log('container', document.getElementById('visContainer').parentNode)
+      })
+      .catch(error => {
```

Comment:
Are we expecting errors? If so we should document here. It seems like we're doing this because errors will fail us out skip the done() call?